notebooks/templates/ai_platform_notebooks_template.ipynb

```diff
@@ -1,524 +1,634 @@
 {
```

Comment:
<p>All of the examples are TF-specific (which is fine), although template users may be using other frameworks. Should we say something like: "for example, when using TensorFlow:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Existing cell. I added your suggestion.</p>